bears/vcs/git/GitCommitBear.py

```diff
@@ -38,6 +39,10 @@ def run(self,
         :param force_body:                 Whether a body shall exist or not.
         :param allow_empty_commit_message: Whether empty commit messages are
                                            allowed or not.
+        :param shortlog_trailing_period:   Whether a dot shall be enforced at
+                                           the end of the shortlog line.
+                                           Providing ``None`` means
```

Comment:
nonononono! If the user provides `None` literally the bear won't execute because `None` isn't bool convertible. Rather say `Provide no value`